src/components/SoundBox/ColumnOfBoxes/SimpleBox/SimpleBox.js

```diff
@@ -5,13 +5,8 @@ import SandButton from './SandButton';
 import VioletButton from './VioletButton';
 
 class SimpleBox extends Component {
-  constructor(props) {
-    super(props);
-    this.state = { clicked: false };
-  }
   render() {
-    const { clicked } = this.state;
-    const { color } = this.props;
+    const { color, clicked, onClick } = this.props;
     let ColorButton;
     switch (color) {
```

Comment:
I think you can also refactor this method and it is so unnecessary to be here. Let's refactor this code and remove this switch at all? What do you think if we will pass the component directly instead of color? cause we use it only to choose here? maybe we do not need this property at all. Let's use render props Pattern.